WiserTaskScheduler/WiserTaskScheduler/Modules/Body/Models/BodyPartModel.cs

```diff
@@ -14,6 +14,23 @@ public class BodyPartModel
         /// </summary>
         [Required]
         public string Text { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the item-id of the used template
+        /// </summary>
+        public ulong ItemId { get; set; }
```

Comment:
I don't think the name `ItemId` is very descriptive here. It doesn't really describe what sort of item it needs to point to. Maybe change it to something like `TemplateItemId` (or even `WiserTemplateItemId`) to better describe what it's used for.